docs/src/components/ReferenceTable/PropsReferenceAccordion.tsx

```diff
@@ -222,44 +229,34 @@ export async function PropsReferenceAccordion({
                       </Link>
                     </DescriptionList.Details>
                   </DescriptionList.Item>
-
                   {PropDescription != null && (
                     <DescriptionList.Item>
-                      <DescriptionList.Separator className="max-xs:pt-2">
-                        <DescriptionList.Term>Description</DescriptionList.Term>
-                      </DescriptionList.Separator>
+                      <DescriptionList.Term className="max-xs:pt-2">
+                        Description
+                      </DescriptionList.Term>
                       {/* one-off override of the default mt/mb on CodeBlock.Root */}
                       <DescriptionList.Details className="[&_[role='figure']]:mt-1 [&_[role='figure']]:mb-1">
                         <PropDescription />
                       </DescriptionList.Details>
                     </DescriptionList.Item>
                   )}
-
```

Comment:
Avoid blank lines in JSX; those always feel arbitrarily placed with no meaning (component separation usually works a lot better to convey context).